modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -1923,6 +1936,148 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
     }
   }
 
+  protected async buildUnsignedSweepTxnTSS(params: RecoverOptions): Promise<OfflineVaultTxInfo | UnsignedSweepTxMPCv2> {
+    const userPublicOrPrivateKeyShare = params.userKey.replace(/\s/g, '');
+    const backupPrivateOrPublicKeyShare = params.backupKey.replace(/\s/g, '');
+
+    const gasLimit = new optionalDeps.ethUtil.BN(this.setGasLimit(params.gasLimit));
+    const gasPrice = params.eip1559
+      ? new optionalDeps.ethUtil.BN(params.eip1559.maxFeePerGas)
+      : new optionalDeps.ethUtil.BN(this.setGasPrice(params.gasPrice));
+
```

Comment:
this is redundant code
we are doing the same here 
https://github.com/BitGo/BitGoJS/pull/5711/files#diff-726ec169a92b0ff7771e86ca7c00a46c13683efa04d26ec3e374514d6256fae4R1899

Reply:
this is not addressed yet